agents/sorenson_dlm/dlm_agent.py

```diff
@@ -319,14 +318,24 @@ def make_parser(parser=None):
                         + "converter ip address")
     pgroup.add_argument('--port', type=int, help="Serial-to-ethernet "
                         + "converter port")
-
+    pgroup.add_argument('--mode', default='acq',
+                        choices=['idel', 'acq', 'acq_reg'])
     return parser
 
 
 if __name__ == '__main__':
     parser = make_parser()
     args = site_config.parse_args(agent_class='DLMAgent',
                                   parser=parser)
+    # Automatically acquire data if request (default)
+    init_params = False
+    if args.mode == 'init':
+        init_params = {'auto_acquire': False,
+                       'acq_params': {},
+                       'configfile': args.configfile}
+    elif args.mode == 'acq':
+        init_params = {'auto_acquire': True,
+                       'acq_params': {}}
```

Comment:
This looks good, but then this `init_params` dict needs to be passed to the `init_dlm` task when it is registered on line 364. It should be passed to the `startup` argument.